### PR TITLE
chore: Fix failed test cases on github actions environment

### PR DESCRIPTION
**Pull request by bilalawan321**
[LEARNER-9434](https://2u-internal.atlassian.net/browse/LEARNER-9434)

**Description**

- Fix all test cases that are failing on the GitHub actions environment
- Put whats new test cases under the flag of enable workflows
- Comment out register screen test cases that require unique IDs from the dev team
- Comment out edit profile test cases after COPPA compliance

### CODE REVIEW

**Review by moeez96 — COMMENTED** [review]
Please remove commented Code, It is a good practice to not have any commented code in the live repo.
In the future, if we want to add some code back, we can always refer to Github history for that file and add it in.
Please request for review after implementing this change.

File: tests/android/tests/test_android_course_subsection.py [hunk 1]
```diff
@@ -76,13 +76,14 @@ def test_subsection_elements_smoke(self, set_capabilities, setup_logging):
...
-        navigation_icon = android_course_dashboard_page.get_navigation_icon()
-        assert navigation_icon.get_attribute('content-desc') == strings.COURSE_DASHBOARD_NAVIGATION_ICON
-        assert android_course_dashboard_page.get_all_text_views()[0].text in course_video_content
+        # commenting this test case until we get the id from dev team on this element
+        # set_capabilities.back()
+        # course_video_content = android_course_section_page.get_course_video_row().text
+        # android_course_section_page.get_course_video_row().click()
+        # navigation_icon = android_course_dashboard_page.get_navigation_icon()
+        # assert navigation_icon.get_attribute('content-desc') == strings.COURSE_DASHBOARD_NAVIGATION_ICON
+        # assert android_course_dashboard_page.get_all_text_views()[0].text in course_video_content
```
[the remark this answers is not in the document]
Comment (bilalawan321) [reply tests/android/tests/test_android_course_subsection.py]: @moeez96 I have removed commented code from all files and PR is ready for review again.

File: tests/android/tests/test_android_edit_profile.py [hunk 2]
```diff
@@ -140,22 +62,23 @@ def test_update_profile_location(self, set_capabilities, setup_logging):
...
         android_profile_screen = AndroidProfile(set_capabilities, setup_logging)
 
-        if (edit_profile_screen.get_by_class_from_elements(
-                android_elements.all_textviews,
-                global_contents.seventh_existence).get_attribute('enabled') == 'false'):
-            edit_profile_screen.get_element_by_id(android_elements.edit_profile_full_view).click()
+        if global_contents.enable_workflows is False:
+            if (edit_profile_screen.get_by_class_from_elements(
+                    android_elements.all_textviews,
+                    global_contents.seventh_existence).get_attribute('enabled') == 'false'):
```
Comment (moeez96) [inline tests/android/tests/test_android_edit_profile.py:68]: Can we reduce the lines from this if condition by assigning the conditional value to a variable and using the variable inside the condition itself? >> Better readability

**Review by moeez96 — APPROVED** [review]
Approved with a comment

File: tests/android/tests/test_android_edit_profile.py (hunk 2 above)
Comment (bilalawan321) [reply tests/android/tests/test_android_edit_profile.py]: Fixed.

**Pull request merged**